Documentation/guides/about_the_dangerfile.html.md

```diff
@@ -90,7 +90,8 @@ let package = Package(
 ...
 products: [
     ...
-    .library(name: "DangerDeps[Product name (optional)]", type: .dynamic, targets: ["DangerDependencies"]), // dev
+    // Library name must be `DangerDeps` otherwise `Danger` won't be able to import it.
+    .library(name: "DangerDeps", type: .dynamic, targets: ["DangerDependencies"]), // dev
```

Comment:
```suggestion
    // Library name must start with `DangerDeps` otherwise `Danger` won't be able to import it.
    .library(name: "DangerDeps", type: .dynamic, targets: ["DangerDependencies"]), // dev
```